crates/sui-config/src/local_ip_utils.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
This is a rename of utils.rs, but also with consolidated ip management for both simtest and non-simtest